packages/@aws-cdk/cdk-deploy/lib/cdk-toolkit.ts

```diff
@@ -1,11 +1,10 @@
+import { data, deserializeStructure, error, highlight, print, success } from '@aws-cdk/cdk-common';
```

Comment:
This class feels like it should go in the common library (or stay in the main toolkit)